tests/system/action/speaker/test_create_point_of_order.py

```diff
@@ -0,0 +1,195 @@
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "speaker.create",
+                    "data": [
+                        {
+                            "user_id": 9,
+                            "list_of_speakers_id": 23,
+                            "point_of_order": True,
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "speaker/3",
+            {
+                "user_id": 9,
+                "list_of_speakers_id": 23,
+                "weight": 10000,
+                "point_of_order": None,
+            },
+        )
+        self.assert_model_exists(
+            "speaker/4",
+            {
+                "user_id": 9,
+                "list_of_speakers_id": 23,
+                "weight": 9999,
+                "point_of_order": True,
+            },
+        )
+        list_of_speakers = self.get_model("list_of_speakers/23")
+        self.assertListEqual(list_of_speakers["speaker_ids"], [1, 2, 3, 4])
+        user = self.get_model("user/9")
+        self.assertListEqual(user["speaker_$7844_ids"], [3, 4])
+        self.assertListEqual(user["speaker_$_ids"], ["7844"])
+
+    def test_create_regular_besides_poo(self) -> None:
+        self.create_model("meeting/7844", {"name": "name_asdewqasd"})
+        self.create_model("user/7", {"username": "talking"})
+        self.create_model("user/8", {"username": "waiting"})
+        self.create_model(
+            "user/9",
+            {
+                "username": "waiting and poo",
+                "speaker_$7844_ids": [3],
+                "speaker_$_ids": ["7844"],
+            },
+        )
+        self.create_model(
+            "speaker/1", {"user_id": 7, "list_of_speakers_id": 23, "begin_time": 100000}
+        )
+        self.create_model(
+            "speaker/2", {"user_id": 8, "list_of_speakers_id": 23, "weight": 10000}
+        )
+        self.create_model(
+            "speaker/3",
+            {"user_id": 9, "list_of_speakers_id": 23, "point_of_order": True},
+        )
+        self.create_model(
+            "list_of_speakers/23", {"speaker_ids": [1, 2, 3], "meeting_id": 7844}
+        )
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "speaker.create",
+                    "data": [{"user_id": 9, "list_of_speakers_id": 23}],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
+            "speaker/3",
+            {"user_id": 9, "list_of_speakers_id": 23, "point_of_order": True},
+        )
+        self.assert_model_exists(
+            "speaker/4",
+            {"user_id": 9, "list_of_speakers_id": 23, "point_of_order": None},
```

Comment:
Maybe just leave the key `point_of_order` out.

Reply:
Done